translator/jsonconfig/mergeJsonConfig.go

```diff
@@ -16,20 +17,29 @@ import (
 	"github.com/aws/amazon-cloudwatch-agent/translator/util"
 )
 
-func MergeJsonConfigMaps(jsonConfigMapMap map[string]map[string]interface{}, defaultJsonConfigMap map[string]interface{}, multiConfig string) (map[string]interface{}, error) {
+func MergeJsonConfigMaps(jsonConfigMapMap map[string]map[string]interface{}, defaultJsonConfigMap map[string]interface{}, multiConfig string, strictValidation bool, inputJsonDirPathErr error ) (map[string]interface{}, error) {
 	if jsonConfigMapMap == nil || len(jsonConfigMapMap) == 0 {
 		if os.Getenv(config.USE_DEFAULT_CONFIG) == config.USE_DEFAULT_CONFIG_TRUE {
 			// When USE_DEFAULT_CONFIG is true, ECS and EKS will be supposed to use different default config. EKS default config logic will be added when necessary
 			if ecsutil.GetECSUtilSingleton().IsECS() {
-				fmt.Println("No json config files found, use the default ecs config")
+				fmt.Println("Invalid agent's json config files, use the default ecs config")
 				return util.GetJsonMapFromJsonBytes([]byte(config.DefaultECSJsonConfig()))
 			}
 		}
-		if multiConfig == "remove" {
-			fmt.Println("No json config files found, please provide config, exit now")
-			os.Exit(config.ERR_CODE_NOJSONFILE)
+
+		if errors.Is(inputJsonDirPathErr, os.ErrNotExist) {
+			//When there is no input from flag --input, --input-dir and cannot find agent's config through containerized environment
+			log.Printf("No agent's json config was found from containerized environment or file path.")
```

Comment:
I'm not sure we _need_ to specify about containerized environments. What is the reasoning behind including that in the log?

Reply:
We aren't including any information about what the error is. Would we be able to distinguish between the file not existing on a host versus not being accessible in a container? If not then I think changing this log just makes it more confusing.